tools/toolchains/gcc.py

```diff
@@ -256,8 +256,6 @@ def link(self, output, objects, libraries, lib_dirs, mem_map):
         # Exec command
         self.notify.cc_verbose("Link: %s" % ' '.join(cmd))
         self.default_cmd(cmd)
-        if self.target.core == "Cortex-M23" or self.target.core == "Cortex-M33":
```

Comment:
Why is this output no longer needed? (Commit only mentioned something about a build issue)

Reply:
Code for creation of ‘secure_file’ was initially in this section and later moved to other part. Variable ‘secure_file’ is not available here and hence throws error